addon/linux_esdm_es/esdm_es_irq.c

```diff
@@ -682,12 +682,14 @@ static void esdm_irq_es_state(unsigned char *buf, size_t buflen)
 
 	/* Assume the esdm_drng_init lock is taken by caller */
 	snprintf(buf, buflen,
-		 " Hash for operating entropy pool: %s\n"
-		 " Available entropy: %u\n"
-		 " per-CPU interrupt collection size: %u\n"
-		 " Standards compliance: %s\n"
-		 " High-resolution timer: %s\n"
-		 " Continuous compression: %s\n",
+		 "{\n"
```

Comment:
This is how I propose it - no JSON-C. Unless you really think JSON-C is indispensable, I would like to recommend this approach -then we do not need to have a configure option.